main.py

```diff
@@ -179,72 +313,42 @@ def main(cl_arguments):
                 "GPU access failed. You might be using a CPU-only installation of PyTorch. Falling back to CPU.")
             args.cuda = -1
 
-    # Prepare data #
+    return args, seed
+
+def main(cl_arguments):
```

Comment:
Try to do a bit of cleanup to the comments that define the major sections of main—they don't seem especially consistent.

Reply:
Mind being consistent about whitespace before/after the comments that introduce new sections of the code?